docs/calendar.md

```diff
@@ -220,6 +220,7 @@ cal = Temporal.Calendar.from('gregory');
 cal = Temporal.Calendar.from('2020-01-13T16:31:00.065858086');
 cal = Temporal.Calendar.from('2020-01-13T16:31:00.065858086-08:00[America/Vancouver][u-ca=iso8601]');
 /* WRONG */ cal = Temporal.Calendar.from('[u-ca-iso8601]'); // => throws, lone annotation not a valid ISO 8601 string
+cal = Temporal.Calendar.from('202001'); // OK because YYYYMM is ISO format of YearMonth
```

Comment:
I would recommend omitting this line, because while it will not throw and will return the ISO 8601 calendar, it's also a confusing example. 

We don't show other formats for `Temporal.Calendar.from` for the same reason. 

Could you remove this line please?  Thanks!

Your other changes look great. Once this line is removed, we can approve and merge this PR. Thanks for the PR!